src/screens/Donate/Donate.style.tsx

```diff
@@ -113,3 +140,7 @@ export const SectionHeader = styled.h2`
   font-weight: bold;
   line-height: 1.4;
 `;
+
+export const StyledIframe = styled.iframe`
+  border: 0;
```

Comment:
why nixing the border? the iframe looks pretty floaty/un-contained without a border

Reply:
No border is just the default style for GiveMomentum. With a border, it current looks like this. I'll play around with it and see if I can style it similar to the GiveButter embed.
<img width="447" alt="image" src="https://user-images.githubusercontent.com/46338131/177435582-e5daec7a-4fde-44a6-884e-1ee6ec11d031.png">